### ISSUE TITLE
Add documentation for GSuite SSO

### ISSUE THREAD

**Issue by arianvp** [wireapp/wire-docs#40]
We tested it internally and it works fine. We should document it :)

**Comment by arianvp** [comment]
@taneli-wire took some notes and pictures while we did it. would you be able to share these somewhere so we can add them to the docs?

**Comment by taneli-wire** [comment]
Yes, will do. What's the best way to get this documented. Should I make instructions in a Google Doc and share that with you?

**Comment by arianvp** [comment]
That's okay with me.  

If you feel adventurous, you could also try cloning this repository and make the changes directly. but I'm not sure how comfortable you would be with that.
